roles/repo_setup/tasks/artifacts.yml

```diff
@@ -17,22 +17,22 @@
   environment: "{{ cifmw_repo_setup_env | default({}) }}"
   register: _get_hash
 
-- name: Dump full hash in delorean.repo.md5 file
-  ansible.builtin.copy:
-    content: |
-      {{ _get_hash.stdout | from_json | community.general.json_query('full_hash') }}
-    dest: "{{ cifmw_repo_setup_basedir }}/artifacts/repositories/delorean.repo.md5"
+- name: Save repo-setup hash content
+  vars:
+    _repo_setup_json: "{{ _get_hash.stdout | from_json }}"
+  block:
+    - name: Dump full hash in delorean.repo.md5 file
+      ansible.builtin.copy:
+        content: |
+          {{ _repo_setup_json['full_hash'] }}
+        dest: "{{ cifmw_repo_setup_basedir }}/artifacts/repositories/delorean.repo.md5"
 
-- name: Store the needed data
-  ansible.builtin.set_fact:
-    _hash: "{{ _get_hash.stdout }}"
-
-- name: Export hash for further use
-  ansible.builtin.set_fact:
-    cifmw_repo_setup_full_hash: "{{ _hash['full_hash'] }}"
-    cifmw_repo_setup_commit_hash: "{{ _hash['commit_hash'] }}"
-    cifmw_repo_setup_distro_hash: "{{ _hash['distro_hash'] }}"
-    cifmw_repo_setup_extended_hash: "{{ _hash['extended_hash'] }}"
-    cifmw_repo_setup_dlrn_api_url: "{{ _hash['dlrn_api_url'] }}"
-    cifmw_repo_setup_release: "{{ _hash['release'] }}"
-    cacheable: true
+    - name: Export hashes facts for further use
+      ansible.builtin.set_fact:
+        cifmw_repo_setup_full_hash: "{{ _hash['full_hash'] }}"
```

Comment:
this should be `_get_hash['full_hash']` or put back the `Store the needed data` task, if I'm reading correctly